_docs/setup/kubernetes/helm.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Istio Helm Chart Instructions
+overview: Instructions for the setup and configuration of Istio using the Helm package manager.
+
+order: 30
+
+layout: docs
+type: markdown
+---
+
+{% include home.html %}
+
+Quick Start instructions for the setup and configuration of Istio using the Helm package manager.
+
+
+## Prerequisites
+
+The following instructions require you have access to Helm **2.7.2 or newer** in your Kubernetes environment or 
+alternately the ability to modify RBAC rules required to install Helm.  Additionally Kubernetes **1.7.3 or newer**
+is also required.
+
+## Deploy with Helm
+
+1. If a service account has not already been installed for Helm, please install one:
+  ```bash
+  kubectl create -f install/kubernetes/helm-service-account.yaml
+  ```
+
+1. Initialize Helm:
+  ```bash
+  helm init --service-account tiller
+  ```
+
+1. Create the Helm chart:
+  ```bash
+  helm install install/kuberenetes/helm/istio --name istio
```

Comment:
ping ^ kuberenetes
can you try the command line too / does it work :-)

Reply:
A sign I need to go to bed :-) - I have run this workflow about 10-15 times in the last 2 days, however, I was manually typing and using completion.  I'll make the typo change, render the docs, and C&P the docs into a log - in the morning :)  Thanks for the patience.

Reply:
you can use the "copy" button from the rendered version and paste in terminal to be sure it's the right command :-) 

thx !

Reply:
yup - my 10-15 runs were for training purposes - so a CP isn't ideal in that model.